test/cmocka/src/math/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 
 add_subdirectory(numbers)
 add_subdirectory(trig)
+add_subdirectory(arithmetic)
```

Comment:
Need to make this a Kconfig option as it will increase code size i.e. processing that use it will enable it in kconfig.

Reply:
Won't linker drop it since it won't be used/referenced?

Reply:
What kind modification/change is expected ? Thank you!!

Reply:
Updated as per suggestion from @singalsu, disabled base2log for `CHT`, `BYT`, `BDW` platform [ XCC and GCC ]